web/src/bootstrap_typeahead.ts

```diff
@@ -75,12 +75,6 @@
  *   If typeahead would go off the top of the screen, we set its top to 0 instead.
  *   This patch should be replaced with something more flexible.
  *
- * 7. Ignore IME Enter events:
```

Comment:
Should you renumber the other points if you're removing 7?

Reply:
I didn't in case there was some reference to these points in the commit messages or in the code.

Reply:
This section is probably of decreasing value now that we've forked it so much anyway. But 103b8f6de3090810e36c2e9b7da61c2e4c0cd299 does confirm that this is all it did.